### ISSUE TITLE
64 bit required for App Store submissions

### ISSUE THREAD

**Issue by CoIin** [alebianco/ANE-Google-Analytics#61]
As you no doubt know, from Feb 1st new apps have to have 64 bit compatible binaries. Are you working on a 64 bit compatible ANE?

**Comment by alebianco** [comment]
Yes, there will be 64-bit support but I can't guarantee an ETA at the moment.

The next milestone is being built with 64-bit support from the beginning. I did some early test with AIR 16 beta already. I'm a bit disappointed at how flaky adobe's integration is but they're making progress.

The current release could possibly receive the same upgrade, depending on how much work it is and how late the next milestone will come.

**Comment by wanting252** [comment]
Google Analytics ANE is also my delay stone to publish my game. Hope to hear a good news from you soon.

**Comment by harry248** [comment]
Thanks for your great work, your ANE works like a charm. I hope you can deliver a 64-bit update soon!

**Comment by lucien144** [comment]
:+1:

**Comment by ZwickTheGreat** [comment]
+1

**Comment by johnd09** [comment]
+1

**Comment by alebianco** [comment]
Almost done guys.
I just need to upgrade google's native library because the current one doesn't support 64bit. this might cause some issues so finger crossed. i should have an update in the next couple of days

**Comment by fix89** [comment]
+1

**Comment by wanting252** [comment]
I go to this site everyday to see the update :))

**Comment by wanting252** [comment]
Any updates for 64 bit, sir?

**Comment by leossmith** [comment]
:+1: cant wait :)

**Comment by mogue2k3** [comment]
Getting nervous!!!

**Comment by alebianco** [comment]
I think I owe an update here ...

It's taking longer than expected because the native sdks have changed significantly in the meantime and i need to upgrade them in order to support 64bit. that required quite a lot of changes on the native side of the extension but i managed to leave the as3 interfaces untouched.

The code update is done, and the extension compiles ... but that's no guarantee.
I'm doing a first round of tests to make sure there's no stupid runtime error waiting to happen.
I'm going to release it by the end of the week and ask everyone to do a deeper round of testing to see that everything works as expected.

We're almost there guys ... bear with me and my slowlyness

**Comment by mogue2k3** [comment]
Yay!

I know you're working hard.
You're going to make us very happy when the new version comes out and we can check that everything works perfectly.

Thank you for your work.

**Comment by laan** [comment]
just thanks for author! well job!

**Comment by leossmith** [comment]
thanks :+1:  awesome work!

**Comment by sfcal99** [comment]
great job! May I know where is the extension you compile?

**Comment by chadupton** [comment]
Thank you! Looking forward to it.

**Comment by wanting252** [comment]
It's nearly 1st Feb limit day from Apple. Are there any new updates, sir? :)

**Comment by longngo78** [comment]
Can't wait :)

**Comment by wanting252** [comment]
Don't know should I replace this ANE or not :(. Love Google Analytics, but cannot wait so long for upload new version :(

**Comment by shivanno** [comment]
I love your work, this is the last 64 bit ANE i need ( no presure of course ) X)     :+1:

**Comment by laan** [comment]
It is too long. So said.

**Comment by wanting252** [comment]
@Iaan Just replace this ANE by Flox to upload to App Store. Sadly I cannot delay my projects too long :(

**Comment by laan** [comment]
@wanting252  Thanks for your reply. But what is Flox? Would you please give a detail page? Thanks again.

**Comment by harry248** [comment]
If you want to pay you can also use the Analytics ANE from Milkman Games: http://www.milkmangames.com/blog/tools/#ganalytics

**Comment by gnomjogson** [comment]
@Iaan http://lmgtfy.com/?q=flox+gamea

**Comment by laan** [comment]
@harry248  milkmanagames‘s ANE support 64-bit system?

**Comment by harry248** [comment]
Their GoViral and CoreMobile ANE's are 64-bit compatible since a few weeks now, so I expect their Analytics ANE to be compatible too. Haven't purchased this one yet.

**Comment by laan** [comment]
@harry248  So, there is no Google Analytics ANE supports 64-bit system.

**Comment by harry248** [comment]
As I said - I don't know, because I did not purchase this extension yet. I purchased other extensions from Milkman Games and those are already 64bit compatible. You will have to ask them but I expect the analytics extension to be compatible too.

**Comment by jazzjoe** [comment]
all milkman ane is 64 bit compatible atm. however if you could use the Alebianco's ANE for free with the same function, that would be great. I could wait for a little more, but in the mean time I suggest you guys try Flurry instead of GA. 64 bit version of Flurry could be found in freshplanet

**Comment by djnr** [comment]
+1

**Comment by hassanMak** [comment]
Thanks for your great work. 
I hope you can deliver a 64-bit update soon!

**Comment by ZwickTheGreat** [comment]
Any update? Should I wait? :)

**Comment by halfjust** [comment]
+1

**Comment by mingoes** [comment]
Looking forward!

**Comment by leossmith** [comment]
any news on this?

**Comment by Matan** [comment]
I'd suggest buying the milkmangames extension, I did, it works perfectly for iOS x64 and Android.

**Comment by coppepam** [comment]
+1

**Comment by ghost** [comment]
I need 64 bit ANE! Please!

**Comment by fix89** [comment]
news?

**Comment by vicker** [comment]
+1

Thumbs up for the great work.

**Comment by honestdollaricloud** [comment]
I know it is too much to ask since it's free, but could you at least let us know if you plan to update or not? and an ETA on this? just to see if decide to wait or get another solution... thanks and we really appreciate your work!

**Comment by djnr** [comment]
I know it is too much to ask since it's free, but could you at least let us know if you plan to update or not? and an ETA on this? just to see if decide to wait or get another solution... thanks and we really appreciate your work!

**Comment by leossmith** [comment]
i already have gone with milkmangames ane and it works perfectly. unfortunately i had to drop this.

**Comment by alanlanglois** [comment]
+1

**Comment by SlavomirDurej** [comment]
any chance you would share a ANE binaries that are compatible with the newest iOS8 apple requirements ? Even if not everything work perfectly ?
Cheers!
